script.js

```diff
@@ -1,4 +1,160 @@
+/*Create a version of the Beat That dice game, where players create the largest number they can by concatenating random dice roll digits:
+
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.*/
+
+var currentPlayer = "";
+var currentGameStatus = "player 1 turn";
+var player1Roll = [];
+var player2Roll = [];
+var totalGames = 0;
+var player1Win = 0;
+var player2Win = 0;
+var player1Numbers = [];
+var player2Numbers = [];
+
+// create a function that stores both player's roll and outputs them
+var generatePlayerRoll = function () {
+  var allPlayerRoll = [];
+
+  for (var counter = 0; counter < 2; counter += 1) {
+    var playerRoll = [];
+
+    var diceRoll = rollDice();
+    playerRoll.push(diceRoll);
+
+    // ensure second roll is not the same as the first roll
+    diceRoll = rollDice();
+    while (diceRoll == playerRoll[0]) {
+      diceRoll = rollDice();
+    }
+    playerRoll.push(diceRoll);
+
+    allPlayerRoll.push(playerRoll);
+  }
+
+  player1Roll = allPlayerRoll[0];
+  player2Roll = allPlayerRoll[1];
+};
```

Comment:
Great job using a loop instead of manually coding this bit of code! As mentioned yesterday, you can consider storing the numOfPlayers as a global variable and using that instead of 2 as the condition for the loop.

This will make it easier to change your code when you want to allow a variable number of players.